### PR TITLE
AlertManager and Debug ability

### PR DESCRIPTION
**Pull request by aneeshkp**
1. Added integration with Prometheus AlertManager and exposure of Event API to post back to SmartGateway. 
2. Added ability to enable and disable debug  from config file or command line, with ability to enable debug from args without changing config file.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -173,6 +173,9 @@ following command to get the vendored dependencies.
 
 
     $ go run metrics.go --config sa.metrics.config.json
+### With debug`
```
Comment (leifmadsen) [inline README.md]: Remove the trailing backtick

File: amqp/sender.go [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License.
+*/
+
+package amqp10
+
+import (
+	//	"flag"
```
Comment (leifmadsen) [inline amqp/sender.go]: if you're not using it, delete it

File: amqp/sender.go [hunk 3]
```diff
@@ -0,0 +1,136 @@
...
+	"log"
+	"os"
+	"strings"
+
+	"qpid.apache.org/amqp"
+	"qpid.apache.org/electron"
+	//  "reflect"
+)
+
+// Usage and command-line flags
```
Comment (leifmadsen) [inline amqp/sender.go]: Remove blocks of code that are commented out

File: amqp/sender.go (hunk 2 above)
Comment (aneeshkp) [reply amqp/sender.go]: Done

File: README.md (hunk 1 above)
Comment (aneeshkp) [reply README.md]: Done

File: amqp/sender.go (hunk 3 above)
Comment (aneeshkp) [reply amqp/sender.go]: done

**Pull request merged**